lib/foodcritic/rules/fc084.rb

```diff
@@ -0,0 +1,9 @@
+rule "FC084", "Deprecated Chef::REST class used" do
+  tags %w{chef13 deprecated}
+  def rest(ast)
+    ast.xpath('//const_path_ref/const[@value="REST"]/..//const[@value="Chef"]/../../..')
```

Comment:
Might be easier to say something like this `//const_path_ref[const[@value="Chef"]][const[@value="REST"]]`? Not sure that's exactly equivalent though.